### DISCUSSION TITLE
Multiple event triggers from bulk update

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#14813] [General]
This is a hypothetical question as I haven’t tested it yet. Suppose I have an event rule that triggers when a device is modified. Then I run a script that modifies 1000 devices all at once. Will the event trigger run 1000 times? That might be desirable in some cases, but for my use I would like it to trigger only once. Is there some way to make it wait some time (15 seconds) to see if the same trigger happens again, and if so only run the latest?

**Comment by abhi1693** [comment]
Events are by objects and if you make 1000 changes, you'll have 1000 events
